tests/Unit/Routing/PagedRouteCollectionTest.php

```diff
@@ -47,12 +47,16 @@ public function testIterator($amountRoutes, $routesLoadedInParallel, $expectedCa
         }
         $names = array_keys($routes);
 
+        $with = [];
+        $will = [];
         foreach ($expectedCalls as $i => $range) {
-            $this->routeProvider->expects($this->at($i))
-              ->method('getRoutesPaged')
-              ->with($range[0], $range[1])
-              ->will($this->returnValue(array_slice($routes, $range[0], $range[1])));
+            $with[] = [$range[0], $range[1]];
+            $will[] = array_slice($routes, $range[0], $range[1]);
         }
+        $this->routeProvider->expects($this->exactly(\count($expectedCalls)))
```

Comment:
sorry, i was wrong and you were right :man_facepalming: - can you please restore the temporary variable again, as $this->routeProvider is the proxy, and not the method mocking thingy?